html/css/custom.css

```diff
@@ -41,11 +41,10 @@ p.release {
 .site-header {
   background-color: black;
   background-image: url(../img/background13.jpg);
```

Comment:
We need to update this. Plus bump the version on the import of this file. The background here has moved to a new location.